SwiftStaticLibrary/SampleObjcClassFromStaticLibrary.m

```diff
@@ -0,0 +1,23 @@
+//
+//  SampleObjcClassFromStaticLibrary.m
```

Comment:
Objective-C code in a static library with mixed code.  This is working as expected.